frontend/src/metabase/search/components/InfoText/InfoText.unit.spec.tsx

```diff
@@ -131,7 +131,7 @@ describe("InfoText", () => {
         `/collection/${MOCK_COLLECTION.id}-collection-name`,
       );
 
-      expect(screen.getByTestId("revision-history-button")).toHaveTextContent(
+      expect(screen.getByTestId("revision-history-text")).toHaveTextContent(
```

Comment:
`InfoText` is only used by search UI, so no `onClick` is passed.

![Screenshot 2024-10-15 at 4 20 11 PM](https://github.com/user-attachments/assets/72d720fa-3fdd-46f0-b4ac-6319b623e3a0)